deploy/tf-module-server/resources.yaml

```
@@ -15,7 +15,7 @@ spec:
         app.kubernetes.io/name: tf-module-server
     spec:
       containers:
-      - image: nginx:1.25
+      - image: ghcr.io/radius-project/mirror/nginx:1.25
```

MCR has a published image available. We could use that so we don't have to worry about maintaining this image (assuming tests pass) 
```suggestion
      - image: mcr.microsoft.com/azurelinux/base/nginx:1.25
```